src/System.Net.Requests/ref/System.Net.Requests.cs

```diff
@@ -92,12 +92,51 @@ public partial class HttpWebRequest : System.Net.WebRequest
         public override System.Net.WebResponse EndGetResponse(System.IAsyncResult asyncResult) { return default(System.Net.WebResponse); }
+        public override System.Net.WebResponse GetResponse() { return default(System.Net.WebResponse); }
+        public virtual bool AllowWriteStreamBuffering { get { return default(bool); } set { } }        
+        public virtual bool AllowAutoRedirect { get { return default(bool); } set { } }
+        public Uri Address { get { return default(Uri); } }
+        public DecompressionMethods AutomaticDecompression { get { return default(DecompressionMethods); } set { } }
+        public DateTime Date { get { return default(DateTime); } set { } }
+        public static int DefaultMaximumResponseHeadersLength { get { return default(int); } set { } }
+        public static int DefaultMaximumErrorResponseLength { get { return default(int); } set { } }
+        public System.Security.Cryptography.X509Certificates.X509CertificateCollection ClientCertificates {
+            get { return default(System.Security.Cryptography.X509Certificates.X509CertificateCollection); } set { } }
+        public string Expect { get { return default(string); } set { } }
+        public DateTime IfModifiedSince { get { return default(DateTime); } set { } }
+        public bool KeepAlive { get { return default(bool); } set { } }
+        public int MaximumAutomaticRedirections { get { return default(int); } set { } }
+        public int MaximumResponseHeadersLength { get { return default(int); } set { } }
+        public string MediaType { get { return default(string); } set { } }        
+        public System.IO.Stream GetRequestStream(out TransportContext context) { throw null; }        
+        // new
+        public bool Pipelined { get { return default(bool); } set { } }
+        public Version ProtocolVersion { get { return default(Version); } set { } }        
+        public int ReadWriteTimeout { get { return default(int); } set { } }
+        public string Referer { get { return default(string); } set { } }
+        public bool SendChunked { get { return default(bool); } set { } }
+        public ServicePoint ServicePoint { get { return default(ServicePoint); } }       
+        public string TransferEncoding { get { return default(string); } set { } }
+        public string UserAgent { get { return default(string); } set { } }
+        public bool UnsafeAuthenticatedConnectionSharing { get { return default(bool); } set { } }
+        public override System.IO.Stream GetRequestStream() { return default(System.IO.Stream); }
+        public void AddRange(System.Int32 range) { }
+        public void AddRange(System.Int32 from,System.Int32 to) { }
+        public void AddRange(System.Int64 range) { }
+        public void AddRange(System.Int64 from,System.Int64 to) { }
+        public void AddRange(System.String rangeSpecifier, System.Int32 range) { }
```

Comment:
Nit: The rest of this file uses type keywords `int`, `long`, `string`, etc. instead of `Int32`, `Int64`, `String`, etc. It'd be good to be consistent.